examples/eks-cluster-with-observability/data.tf

```diff
@@ -0,0 +1,20 @@
+data "aws_region" "current" {}
+
+data "aws_availability_zones" "available" {}
+
+data "aws_eks_cluster" "cluster" {
+  name = module.aws-eks-accelerator-for-terraform.eks_cluster_id
+}
+
+data "aws_eks_cluster_auth" "cluster" {
+  name = module.aws-eks-accelerator-for-terraform.eks_cluster_id
+}
+
+data "aws_iam_policy_document" "fluentbit-opensearch-access" {
```

Comment:
We should use a more restrictive policy such as https://www.eksworkshop.com/intermediate/230_logging/prereqs/#creating-an-iam-role-and-policy-for-your-service-account.

Reply:
I agree. I've updated the action from `es:*` to `es:ESHttp*`.